src/backend/core/api/viewsets.py

```diff
@@ -599,6 +599,9 @@ def list(self, request, *args, **kwargs):
         # Not calling filter_queryset. We do our own cooking.
         queryset = self.get_queryset()
 
+        # exclude main workspace
+        queryset = queryset.filter(main_workspace=False)
```

Comment:
Why removing the main_workspace from `/items`, even if it is returned by `/me` it seems to make the API less consistent and intuitive as the `/items` route does not return all items, it there a specific another reason?